src/pages/home/report/ReportActionCompose.js

```diff
@@ -24,10 +25,16 @@ const propTypes = {
 
     // The ID of the report actions will be created for
     reportID: PropTypes.number.isRequired,
+
+    isSidebarShown: PropTypes.bool,
+
+    isSmallScreenWidth: PropTypes.bool,
 };
 
 const defaultProps = {
     comment: '',
+    isSidebarShown: true,
```

Comment:
I think in the ReportActionCompose, we should assume that `isSidebarShown` is false, because on mobile under normal circumstances this component is only visible if the sidebar is not.

```suggestion
    isSidebarShown: false,
```